test/tests/rails7.rb

```diff
@@ -13,10 +13,16 @@ def expected
       :controller => 0,
       :model => 0,
       :template => 0,
-      :warning => 22
+      :warning => 23
     }
   end
 
+  def test_ruby_2_7_eol
+    assert_warning check_name: "EOLRuby",
```

Comment:
This will break again on 2023-03-31 (or maybe sooner).

Can you wrap it in code like this? (This is what's done [here](https://github.com/presidentbeef/brakeman/blob/main/test/tests/cves.rb#L351))

```ruby
Date.stub :today, Date.parse('2022-02-10') do
  assert_warning ...
end
```

Reply:
It is updated. I wraped the scann method